include/ozo/core/concept.h

```diff
@@ -471,31 +500,27 @@ auto Handler = [&] (ozo::error_code ec, auto connection) {
 };
  *@endcode
  *
- * `Handler` has to handle an `ozo::error_code` object as first argument, and #Connection implementation
+ * `Handler` has to handle an `ozo::error_code` object as first argument, and `Connection` implementation
```

Comment:
the first argument, and the ... implementation.

Reply:
**Done**